pkg/scanners/scannerv4/nodescan_convert_test.go

```diff
@@ -54,19 +75,56 @@ func (s *indexReportConvertSuite) TestToStorageComponentsOutOfBounds() {
 	in := createOutOfBoundsReport()
 	expectedCVE := &storage.EmbeddedVulnerability{
 		Cve:               "RHSA-2024:4616",
+		Cvss:              7.5,
 		Summary:           "Sample Description",
+		ScoreVersion:      1,
 		SetFixedBy:        &storage.EmbeddedVulnerability_FixedBy{FixedBy: "0:4.16.0-202407111006.p0.gfa84651.assembly.stream.el9"},
 		VulnerabilityType: storage.EmbeddedVulnerability_NODE_VULNERABILITY,
 		Severity:          storage.VulnerabilitySeverity_MODERATE_VULNERABILITY_SEVERITY,
+		CvssV3: &storage.CVSSV3{
+			Vector:              "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+			ExploitabilityScore: 3.9,
+			ImpactScore:         3.6,
+			AttackVector:        2,
+			Availability:        2,
+			Score:               7.5,
+			Severity:            4,
+		},
 	}
 
 	actual := toStorageComponents(in)
 
 	s.Len(actual, 2)
 	for _, c := range actual {
 		// Ensure that each of the components track the expected CVE
-		protoassert.SliceContains(s.T(), c.GetVulns(), expectedCVE)
+		// protoassert.SliceContains(s.T(), c.GetVulns(), expectedCVE)
```

Comment:
Why commented-out?

Reply:
Great spot! That's a debug-time artifact. `Equal` gives a nice diff, whereas `SliceContains` doesn't. Reverted it to its original state.